src/nvim/eval.c

```diff
@@ -250,6 +250,7 @@ static struct vimvar {
   VV(VV__NULL_DICT,       "_null_dict",       VAR_DICT, VV_RO),
   VV(VV__NULL_BLOB,       "_null_blob",       VAR_BLOB, VV_RO),
   VV(VV_LUA,              "lua",              VAR_PARTIAL, VV_RO),
+  VV(VV_CANCEL_EVENT,     "cancel_event",     VAR_BOOL, 0),
```

Comment:
fwiw, I suggested `v:event_cancel` because that helps with discoverability when using and searching for `v:event`.

Reply:
:+1:  I'll change it